web/src/main/java/com/graphhopper/http/cli/ExportNationwideCommand.java

```diff
@@ -0,0 +1,39 @@
+package com.graphhopper.http.cli;
+
+import com.graphhopper.CustomGraphHopperOSM;
+import com.graphhopper.http.GraphHopperManaged;
+import com.graphhopper.http.GraphHopperServerConfiguration;
+import com.graphhopper.replica.StableEdgeIdManager;
+import com.graphhopper.replica.StreetEdgeExporter;
+import io.dropwizard.cli.ConfiguredCommand;
+import io.dropwizard.setup.Bootstrap;
+import net.sourceforge.argparse4j.inf.Namespace;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ExportNationwideCommand extends ConfiguredCommand<GraphHopperServerConfiguration> {
+    private static final Logger logger = LoggerFactory.getLogger(ExportNationwideCommand.class);
+
+    public ExportNationwideCommand() {
+        super("export-nationwide", "Imports nationwide OSM + generates street network CSV file");
+    }
+
+    @Override
+    protected void run(Bootstrap<GraphHopperServerConfiguration> bootstrap, Namespace namespace,
+                       GraphHopperServerConfiguration configuration) {
+        final GraphHopperManaged graphHopper = new GraphHopperManaged(configuration.getGraphHopperConfiguration(), bootstrap.getObjectMapper());
+
+        // Build OSM-only GH graph, collect OSM tag info, and set stable edge IDs (as done in normal import)
```

Comment:
The important distinction between this command and the normal `import` command we always use is that the `import` command writes out all of the parsed OSM info to a DB file, so it can be read in + used in subsequent steps downstream. It turns out that this takes quite a lot of time, and loads of disk space. Removing the write out to disk, which we don't need here (as we're directly using the parsed info right away, so we can keep it in memory) gives us a giant speed improvement